Feed/views.py

```diff
@@ -2,4 +2,4 @@
 
 
 def resumeApp(request):
```

Comment:
I would replace this line with `def Feed(request):` .

Reply:
Replaced it.